tools/generate-transform-jobs/main.go

```diff
@@ -35,7 +35,7 @@ func exit(err error, context string) {
 	os.Exit(1)
 }
 
-var inputDir = flag.String("input-dir", "../config/istio-private_jobs", "directory of input jobs")
+var inputDir = flag.String("input-dir", "prow/config/istio-private_jobs", "directory of input jobs")
```

Comment:
Seems like this code expects to be run from `prow/generate-transform-jobs`.. updated.